eng/pipelines/templates/Static_Source_Analysis.yml

```diff
@@ -71,12 +71,5 @@ steps:
   continueOnError: ${{ parameters.isOfficialBuild }}
   condition: succeededOrFailed()
 
-- task: PublishPipelineArtifact@1
```

Comment:
This stage does not emit a binlog and this task fails if the job fails so I just removed it.